code/modules/transport/pods/vehicle.dm

```diff
@@ -581,8 +581,8 @@
 				SPAWN_DBG(0)
 					M.throw_at(throw_at, movement_controller:velocity_magnitude, 2)
 				logTheThing("combat", src, target, "crashes into [target] [log_loc(src)].")
-				if(isalive(M))
-					vehicular_manslaughter = 0 //we now check if person is alive after hit, if they are dead they get the achievement
+				if(M.health < 0) 
```

Comment:
Wont work as written - health is updated by the health update queue (which runs every half-second, give or take) 😓

Reply:
god damn it

Reply:
Also I think you had the conditional backwards here anyhow :shelterfrog:

Reply:
i'll just take your suggestion of max speed ramming into a human seems less of a pain

Reply:
I don't think that's the case?

Reply:
oh wait nvm, humans don't call their parents